keyboards/dinofizz/fnrow/v1/readme.md

```diff
@@ -0,0 +1,26 @@
+# FnRow v1 
+
+A keyboard featuring a single horizontal row of switches in the style of a "function row" layout.
+
+* Keyboard Maintainer: [Dino Fizzotti](https://www.dinofizzotti.com)
+* [PCB Schematic and layout](https://github.com/dinofizz/fnrow-pcb)
+* [Blog post](https://www.dinofizzotti.com/blog/2021-02-07-fnrow-v1-a-configurable-function-row-layout-mechanical-keyboard/)
+
+##  Features
+
+* USB-C
+* USB ESD and overcurrent protection
+* Dedicated reset switch to enable DFU mode
+* Optional status LED
+
+## Compilation and Flashing
+
+Make compilation example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter bootloader mode?

Reply:
Certainly, thank you for prompting me on this.